thunder.install

```diff
@@ -284,3 +286,31 @@ function thunder_update_8321(): string {
   // Output logged messages to related channel of update execution.
   return $updater->logger()->output();
 }
+
+/**
+ * Add paragraphs and metatag fields to basic page.
+ */
+function thunder_update_8322(): string {
+  $newFields = [
+    'field.field.node.page.field_meta_tags',
+    'field.field.node.page.field_paragraphs',
+  ];
+
+  foreach ($newFields as $newField) {
+    $configDirectory = new FileStorage(\Drupal::service('extension.path.resolver')
+      ->getPath('profile', 'thunder') . '/config/optional');
+    $configRecord = $configDirectory->read($newField);
+    if (!FieldConfig::loadByName($configRecord['entity_type'], $configRecord['bundle'], $configRecord['field_name'])) {
+      FieldConfig::create($configRecord)->save();
+    }
+  }
```

Comment:
You could add this to your update file instead:
```
__global_actions:
  import_configs:
    - field.field.node.page.field_meta_tags
    - field.field.node.page.field_paragraphs
```

Reply:
Why does it not do this by itself?

Reply:
This is at least missing in the documentation :-)